gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -61,7 +65,9 @@
             <property name="allowByTailComment" value="true"/>
             <property name="allowNonPrintableEscapes" value="true"/>
         </module>
-        <module name="AvoidNestedBlocks"/> <!-- Java Coding Guidelines: Avoid nested blocks -->
+        <module name="AvoidNestedBlocks"> <!-- Java Coding Guidelines: Avoid nested blocks -->
```

Comment:
Merrick and Javier are working on bring the docs to OSS. Until then: stash it is.